libs/asset-viewer/src/lib/state/asset-search/asset-search.actions.ts

```diff
@@ -35,4 +36,4 @@ export const closeFilters = createAction('[Asset Viewer] Close Filters');
 export const openResults = createAction('[Asset Viewer] Open Results');
 export const closeResults = createAction('[Asset Viewer] Close Results');
 export const setStudies = createAction('[Asset Viewer] Load Studies', props<{ studies: AllStudyDTOs }>());
-export const loadSearch = createAction('[Asset Viewer] Load Search', props<{ query: AssetSearchQuery }>());
+export const loadSearch = createAction('[Asset Search] Load Search', props<{ query: AssetSearchQuery }>());
```

Comment:
I believe the convetion is to have the top level name be the same for all actions. So theoretically, all actions should be called one or the other (Assert Search or Asset Viewer) or we can split them into separate files